var/spack/repos/builtin/packages/fbgemm/package.py

```diff
@@ -35,4 +35,12 @@ class Fbgemm(CMakePackage):
     depends_on('python', type='build')
     depends_on('llvm-openmp', when='%apple-clang')
 
+    conflicts('%gcc@:4', msg='FBGEMM requires GCC 5+')
```

Comment:
Does this make sure that we are running using GCC5+?

Reply:
Yes, if the user tries to use GCC 4 or older, it will error out and print this message.

Btw, Spack supports several different compilers, so if you know the minimum version of other compilers we can add those too.